org.eclipse.xtext.xbase/src/org/eclipse/xtext/xbase/compiler/XbaseCompiler.java

```diff
@@ -1696,6 +1703,15 @@ protected void appendClosureParameter(JvmFormalParameter closureParam, Lightweig
 		String name = appendable.declareVariable(closureParam, proposedParamName);
 		appendable.append(name);
 	}
+	
+	protected void appendClosureParameterVarArgs(JvmFormalParameter closureParam, LightweightTypeReference parameterType, ITreeAppendable appendable) {
```

Comment:
Add @since here? I know, API tooling is not working, but we should try to indicate API adds anyway, OK?